server/auth/cancel.go

```diff
@@ -116,7 +124,7 @@ func (lo *latestOrders) counts() (total, cancels int) {
 
 	total = len(lo.orders)
 	for _, o := range lo.orders {
-		if o.target != nil {
+		if o.target != nil && o.epochGap >= 0 && o.epochGap < freeCancelThreshold {
```

Comment:
This is the line that matters. This is a rule change. Free cancels after the order sits for one full epoch. If the order is submitted in epoch 10, and the cancel order submitted in epoch 10 or 11, that's a "short" cancel (not a free cancel), and the cancellation can hurt to the user's score.